trieste/acquisition/optimizer.py

```diff
@@ -65,16 +75,25 @@ def _discrete_space(space: DiscreteSearchSpace, target_func: AcquisitionFunction
 
 
 @optimize.register
-def _box(space: Box, target_func: AcquisitionFunction) -> TensorType:
-    trial_search_space = space.discretize(tf.minimum(2000, 500 * tf.shape(space.lower)[-1]))
-    initial_point = optimize(trial_search_space, target_func)
+def optimize_continuous(
+    space: Box, target_func: AcquisitionFunction, num_query_points: int = 1
+) -> TensorType:
+    expanded_space = space ** num_query_points
+    trial_search_space = expanded_space.discretize(
+        tf.minimum(2000, 500 * tf.shape(expanded_space.lower)[-1])
+    )
 
-    bijector = tfp.bijectors.Sigmoid(low=space.lower, high=space.upper)
+    def vectorized_target_func(at):
```

Comment:
vectorized is an easily overloaded term, replace the name with something more obvious? eg `target_func_with_vectorized_inputs`